### ISSUE TITLE
Package announcement

### ISSUE THREAD

**Issue by femtotrader** [oliviermilla/Lucky.jl#13]
Hello,

Maybe you should consider posting here

- https://discourse.julialang.org/c/package-announcements/60
- https://discourse.julialang.org/t/backtesting-framework/47351

Kind regards

**Comment by oliviermilla** [comment]
Yes, it's on my TODO List. :) Thank you very much for understanding this.
